cmd/wallet.go

```diff
@@ -0,0 +1,100 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	walletcmd "github.com/keep-network/keep-core/cmd/wallet"
+	"github.com/keep-network/keep-core/config"
+	"github.com/keep-network/keep-core/pkg/chain/ethereum"
+)
+
+var (
+	walletFlagName       = "wallet"
+	hideSweptFlagName    = "hide-swept"
+	sortByAmountFlagName = "sort-amount"
+)
+
+// WalletCommand contains the definition of tBTC wallets tools.
+var WalletCommand = &cobra.Command{
+	Use:              "wallet",
+	Short:            "tBTC wallets tools",
+	Long:             "The tool exposes commands for interactions with tBTC wallets.",
+	TraverseChildren: true,
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		if err := clientConfig.ReadConfig(
+			configFilePath,
+			cmd.Flags(),
+			config.General, config.Ethereum, config.BitcoinElectrum,
+		); err != nil {
+			logger.Fatalf("error reading config: %v", err)
+		}
+	},
+}
+
+var depositsCommand = cobra.Command{
+	Use:              "deposits",
```

Comment:
Can we rename this command to `list-wallet-deposits`? This will make it more specific.

Reply:
I meant something similar to what I mentioned in https://github.com/keep-network/keep-core/pull/3545#discussion_r1187239035. In this case, `deposits` are also ubiquitous, and making it at least `list-deposits` would make the command clear at first sight, even without referring to its description.

Reply:
Eventually, the wallet coordinator will manage redemptions, moving funds, and sweeps of moved funds. Most likely, the CLI will have to handle them as well. This is why I'm flagging the need to establish a clear and consistent API of the CLI from the beginning. I'm afraid that we won't be able to keep them short in the long term so it seems that it's better to use a predictable pattern from the beginning. For example, sweeping refers to both deposits and moved funds so we can't just have a single 'sweep' command.

Reply:
I would also vote for `list-deposits`. We will have more actions in the coordinator soon and `list-deposits` is in my opinion short enough and unique to not confuse it with anything else.

The actions we'll have to implement in the coordinator:
- list deposits (this PR)
- sweep (#3549)
- increase sweep fee
- list redemptions (will also have to be automated)
- redeem (will also have to be automated)
- increase redemption fee

... then all the actions related to moving funds. I think it would also be nice to allow submitting heartbeat requests via the coordinator CLI.

Reply:
Discussion continued in https://github.com/keep-network/keep-core/pull/3549#discussion_r1185811670